bindings/rust/standard/integration/src/features/pq.rs

```diff
@@ -99,3 +99,77 @@ async fn s2n_server() -> Result<(), Box<dyn std::error::Error>> {
     );
     Ok(())
 }
+
+#[tokio::test]
+async fn s2n_client_mlkem() -> Result<(), Box<dyn std::error::Error>> {
```

Comment:
https://github.com/aws/s2n-tls/pull/5438#discussion_r2237885415 The same naming comment applies to mlkem